pkg/csi/service/wcp/controller.go

```diff
@@ -1456,11 +1458,6 @@ func (c *controller) ControllerGetCapabilities(ctx context.Context, req *csi.Con
 			csi.ControllerServiceCapability_RPC_LIST_VOLUMES_PUBLISHED_NODES)
 	}
 
-	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.BlockVolumeSnapshot) {
```

Comment:
shall we keep this code now and remove when we remove the FSS altogether?

Reply:
There's no point in not reporting the snapshot capabilities after the sidecar is introduced. The csi-snapshotter sidecar checks if the driver is exposing snapshot capabilities during its startup, if no capabilities are exposed then the sidecar terminates, this will show the whole csi driver deployment as failed. Additionally, we can still disable the whole snapshot feature by changing the internal fss at any point since the individual apis are still guarded, this way, we can disable the feature without showing the driver as a failed.